Build issue

## Steps to reproduce

  1. CentOS
  2. ```python setup.py install --headless```

## Observed Results
```
/home/helenjia/h/habitat-sim/src/esp/nav/PathFinder.cpp: In function 'std::tuple<unsigned int, unsigned int, Eigen::Matrix<float, 3, 1, 0, 3, 1> > esp::nav::{anonymous}::projectToPoly(const vec3f&, const dtNavMeshQuery*, const dtQueryFilter*)':
/home/helenjia/h/habitat-sim/src/esp/nav/PathFinder.cpp:42:35: error: converting to 'std::tuple<unsigned int, unsigned int, Eigen::Matrix<float, 3, 1, 0, 3, 1> >' from initializer list would use explicit constructor 'constexpr std::tuple< <template-parameter-1-1> >::tuple(_UElements&& ...) [with _UElements = {unsigned int&, unsigned int&, Eigen::Matrix<float, 3, 1, 0, 3, 1>&}; <template-parameter-2-2> = void; _Elements = {unsigned int, unsigned int, Eigen::Matrix<float, 3, 1, 0, 3, 1>}]'
   return {status, polyRef, polyXYZ};
```

## Solution

* Issue with initializer lists used to initiate a the tuple, in ```line 42``` of ```PathFinder.cpp```
* Can be solved with returning ```return std::make_tuple(args)```
* if it is right I am happy to submit a pull request to fix it

--- a/src/esp/nav/PathFinder.cpp
+++ b/src/esp/nav/PathFinder.cpp
@@ -39,7 +39,7 @@ std::tuple<dtStatus, dtPolyRef, vec3f> projectToPoly(
   dtStatus status = navQuery->findNearestPoly(pt.data(), polyPickExt, filter,
                                               &polyRef, polyXYZ.data());
 
-  return {status, polyRef, polyXYZ};
+  return std::make_tuple(status, polyRef, polyXYZ);
 }
 }  // namespace
 

